framework/reactioncommerce/utils/normalize.ts

```diff
@@ -178,10 +181,10 @@ function normalizeLineItem({
     priceWhenAdded,
     optionTitle,
     variantTitle,
-    quantity,
-  },
-}: CartItemEdge): LineItem {
-  console.log('imageURLs', imageURLs)
+    quantity
+  } = <CartItem>cartItem.node
+
+  console.log('imageURLs', cartItem)
```

Comment:
This was committed by mistake.

Reply:
Is `cartItem.node` really of type `CartItem`? If so, we can leave it in.